CHANGELOG.md

```diff
@@ -45,6 +45,8 @@
   a project contain syntax errors.
 - [Users can opt out of anonymous data gathering.][1328] This can be done with
   the `--no-data-gathering` command-line flag during the startup of the IDE.
+- [Provide a theming API for JavaScript visualizations][1358]. This API can be
+  used to obtain the colors associated with a type or a class GUI elements.
```

Comment:
This should be more user-facing. Like "It is now possible to use Enso theming engine while developing custom visualizations in JavaScript. You can query it for all IDE colors, including colors for a particular type". (please also check it with Ara or Joe (native speakers) to see if its 100% dramatically correct, as CHANGELOG is a public document).